target/linux/ar71xx/base-files/lib/preinit/05_set_preinit_iface_ar71xx

```diff
@@ -17,6 +17,8 @@ set_preinit_iface() {
 	archer-c7 |\
 	bhr-4grv2 |\
 	dir-505-a1 |\
+	e600g |\
+	e600gac |\
```

Comment:
eth1 on these devices is LAN or WAN interface? Better is use LAN interface in preinit/failsafe mode